app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Here is the list of all users</h1>
\ No newline at end of file
+<% if @users.any? %>
+  <% @users.each do |user| %>
+    <a href="/users/<%= user.id %>" class="text-decoration-none">
+    <%= render "shared/user", user: user  %>
+    </a>
```

Comment:
- [ ] [Optional] [Highly recommended] It would be better professionally if you use [link_to method](https://www.rubyguides.com/2019/05/rails-link_to-method/) instead of using regular anchor tag `<a>`.
```suggestion
<%= link_to user_path(user) do %>
   <%= render "shared/user", user: user  %>
<% end %>
```